src/targets/unified.ini

```diff
@@ -130,3 +130,21 @@ upload_command = ${env_common_esp82xx.bf_upload_command}
 
 [env:Unified_ESP32_2400_RX_via_WIFI]
 extends = env:Unified_ESP32_2400_RX_via_UART
+
+[env:Unified_ESP32_900_RX_via_UART]
+extends = env_common_esp32rx, radio_900
+build_flags =
+	${env_common_esp32rx.build_flags}
+	${radio_900.build_flags}
+	${common_env_data.build_flags_rx}
+	-include target/Unified_ESP8285_RX.h
```

Comment:
Time to rename these files to just use ESP?

Reply:
Probably a good idea. I'll do that as part of this PR then...